torchvision/datapoints/_bounding_box.py

```diff
@@ -42,7 +42,7 @@ class BoundingBoxes(Datapoint):
     canvas_size: Tuple[int, int]
 
     @classmethod
-    def _wrap(cls, tensor: torch.Tensor, *, format: BoundingBoxFormat, canvas_size: Tuple[int, int]) -> BoundingBoxes:
+    def _wrap(cls, tensor: torch.Tensor, *, format: BoundingBoxFormat, canvas_size: Tuple[int, int]) -> BoundingBoxes:  # type: ignore[override]
```

Comment:
I didn't try to do this "properly", I only tried to stop `mypy` from being a nuisance. If there's a more correct way, feel free to push.

Reply:
`mypy` is correct here, but I agree that silencing it is the right move here.